test/simplegraphs/generators/staticgraphs.jl

```diff
@@ -536,5 +536,26 @@
         @test_throws DomainError lollipop_graph(0, 1)
         @test_throws DomainError lollipop_graph(-1, -1)
     end
+   @testset "Friendship Graphs" begin
+        # the Friendship Graphs is connected graph consist of n cycle of lenght 3 share only one node 
+        for n in [10, 15, 20]
+         g=friendship_graph(n)
+         @test length(connected_components(g)) == 1
+         @test articulation(g) == [1]
+         cc=biconnected_components(g)
```

Comment:
You seem to to test various properties of your friendship graph. I think though, it would be enough to test the following properties, as they should be enough for a graph to be a friendship graph:
A graph with `m` vertices is a friendship graph, if
* `m` is an odd number
* There is a vertex of degree m - 1
* Every other vertex has degree 2
* There are no self-loops (I am not sure if this condition is even necessary)

Reply:
@SimonDanisch if there is parallel edges we can have false positive, imagine a graph with `2 * n + 1` nodes, one of them has parallel edge with n of the node, the other n nodes represent a simple cycle, this graph will pass that test, but 
i think GL doesn’t support parallel edges am i right? .